pyproject.toml

```diff
@@ -1,3 +1,45 @@
 [build-system]
 # Minimum requirements for the build system to execute (according to PEP 518)
-requires = ["setuptools>=18.0", "wheel", "numpy>=1.10", "cython>=0.23"]
+requires = ['setuptools >= 61.0', 'wheel', 'cython >= 0.23']
+build-backend = 'setuptools.build_meta'
+
+[tool.setuptools]
+packages = ['knossos_utils']
+
+[project]
+name = 'knossos_utils'
+version = '0.1'
+authors = [ {name = 'KNOSSOS team', email='knossosteam@gmail.com'} ]
+description = 'A Python 3.x library for interacting with **KNOSSOS** datasets and annotation files.'
+readme = 'README.md'
+#requires-python = '>=3.8'
```

Comment:
and i actually had that removed (yesterday) and only got it back accidentally

Reply:
I mean, it will be squashed anyway, I guess?